core/storage/trie/impl/persistent_trie_batch_impl.hpp

```diff
@@ -16,18 +17,23 @@ namespace kagome::storage::trie {
   class PersistentTrieBatchImpl : public PersistentTrieBatch {
    public:
     using RootChangedEventHandler = std::function<void(const common::Buffer &)>;
+    enum class Error {
+      ASYNC_OPERATION_FAILED = 1,
```

Comment:
I think it can be renamed or commented on what operation this is, because there's just one async operation.

Reply:
done